### ISSUE TITLE
Feature Request: Option to change/update Lora filename to its title in its Civitai page

### ISSUE THREAD

**Issue by freecoderwaifu** [zixaphir/Stable-Diffusion-Webui-Civitai-Helper#70]
There are too many Loras on Civitai with really unhelpful names, like strings of numbers such as 1702389001733332106.safetensors (that's a real Lora filename by the way), not to mention Loras that have English titles but their filename is in foreign characters. 

This makes them a pain to manage and organize, especially as time goes by and you forget when and where you downloaded them, and what they even do if the thumbnail isn't specific enough.

An option to scan Loras and change their filenames to their titles in their particular Civitai page would be really helpful.

For example, the Lora mentioned above.

1702389001733332106.safetensors

Civitai page:
https://civitai.com/models/262235?modelVersionId=295768

The Lora would be renamed:  Book Conceptual Art.safetensors

**Comment by zixaphir** [comment]
I agree with the sentiment here, but a simple "rename everything" approach has too many pitfalls to be an appropriate solution. For instance, many lora are appropriately named but the civitai page for them functions more like a container for several models that serve different purposes. For instance, the "[age slider](https://civitai.com/models/65214)" set of models all server functionally similar purposes but for different age groups. Having them all be named "age slider.safetensors" would be a step back from their current name. Furthermore, many models are named things that terrible as filenames, having unicode characters that make them pop out when reading them on civitai, but are a headache when dealing with them at a user-interface or filesystem level. Models also contain a variety of languages in their titles that would cause a real headache for people who don't speak those languages, but have the filename already in the language they prefer.

And these are just foreseen issues. I have no idea the number of new issues this could introduce. I highly recommend using the included model downloader that before downloading lets you see the model's downloaded filename and allows you to change it or using the existing rename feature in conjunction with the open model url feature to rename models that have useless filenames, and leaving a comment on the model author's page politely asking them to name their models better.

**Comment by freecoderwaifu** [comment]
Yeah I agree this is something that should be handled first on Civitai's side, by the posters and maybe by the site itself too, encouraging better filenames in general and even better titles.

I've tried the suggestion but it gets less manageable when having too many Loras and mass downloading them. 

I do agree that a mass renamer might be too much though. Maybe a smaller addition to the existing rename button on the cards, a "paste title from .info file" checkbox or button could also be a better option, to handle it individually.
